frappe/public/js/frappe/social/components/ProfileSidebar.vue

```diff
@@ -29,25 +43,42 @@ export default {
 				'owner': this.user_id
 			}
 		}).then(count => {
-			console.log(count);
 			this.post_count = count
+		}),
+		frappe.db.count('Post', {
+			'filters': {
+				liked_by: ['like', '%'+this.user_id+'%']
+			}
+		}).then(count => {
+			this.likes_count = count
 		})
+		
+	},
+	methods: {
+		change_list: function(name){
+			this.$emit('change_list',name)
+		}
 	},
 	computed: {
 		user_avatar() {
 			return frappe.avatar(this.user_id, 'avatar-xl')
 		},
 		user() {
 			return frappe.user_info(this.user_id)
-		}
+		},
+		check_user_perm()
+		{	this.isUser = this.user_id == frappe.session.user? true:false; 
```

Comment:
Move this to data.. as this is a one time check.

Also rename `isUser` to `is_own_profile` 

like

```
is_own_profile = this.user_id === frappe.session.user
```